.env

```diff
@@ -0,0 +1,6 @@
+HOST_IP_1=127
+HOST_IP_2=0
+HOST_IP_3=0
+HOST_IP_4=1
```

Comment:
warum die ip nicht in eins ?
so ist das doch total komisch

Reply:
fixed